gpu/install_gpu_driver.sh

```diff
@@ -96,9 +96,9 @@ readonly -A CUDA_SUBVER=(
           [12.5]="12.5.1"
 )
 
-RUNTIME=$(get_metadata_attribute 'rapids-runtime' 'SPARK')
+RAPIDS_RUNTIME=$(get_metadata_attribute 'rapids-runtime' 'SPARK')
 DEFAULT_CUDA_VERSION='12.4'
-if [[ "${RUNTIME}" != "SPARK" ]]; then
+if [[ "${RAPIDS_RUNTIME}" != "SPARK" ]]; then
   DEFAULT_CUDA_VERSION='11.8'
```

Comment:
If we are using the CUDA 11 build for Spark, this is ok

If we use a CUDA 12 build of Spark, this would be could CUDA 12.2 for RAPIDS 24.08

Reply:
this may be legacy.  I could try removing this conditional block entirely and see how things go...

Reply:
Think we still needed it as Spark-RAPIDS is on a different CUDA version than the rest of RAPIDS

Though ideally we work to get this closer aligned with everything else

For now, am happy leaving it and revisiting in the future

Reply:
There is a different action, spark-rapids, which @nvliyuan maintains where the Spark installer lives.  The existence of reference to spark in this action is generally vestigial

Reply:
Ah ok. Thanks for the context! 🙏

Deferring to you on how best to address 🙂